lib/git_reflow/commands/refresh.rb

```diff
@@ -10,17 +10,14 @@
 arg_name '[remote_location] - remote repository name to fetch updates from (origin by default), [base_branch] - branch that you want to merge with (master by default)'
 command :refresh do |c|
   c.desc 'updates base_branch based on remote and merges the base with your feature_branch'
-  c.arg_name '[remote_location] - remote repository name to fetch updates from (origin by default),
-          [base_branch] - branch that you want to merge with (master by default)'
   c.flag [:r,:remote], default_value: 'origin'
-  c.flag [:b,:branch], default_value: 'master'
+  c.flag [:b,:base], default_value: 'master'
   c.action do |global_options, options, args|
-    # usage: git reflow refresh [remote_location - defaults to origin] [base_branch_name - defaults to master]
+    # usage: git reflow refresh -r [remote_location - defaults to origin] -b [base_branch_name - defaults to master]
```

Comment:
The primary advantage of using a gem like `gli` is that it adds documentation with its syntax.  We have a task to add documentation for the code which is where something like this would be better suited to live.  I'd still like to see the comments removed here.